contrib/qubes/src/fwupd-dom0-update

```diff
@@ -0,0 +1,190 @@
+#!/bin/bash
+
+escape_args() {
+    local eargs=""
+
+    for arg in "$@"; do
+    printf -v eargs '%s%q ' "$eargs" "$arg"
+    done
```

Comment:
does this file have to be shell? I'd be much more comfortable if it was python...

Reply:
I rewrote the shell scripts to the Python.